source/Calamari.Common/Plumbing/IndentedTextWriter.cs

```diff
@@ -476,13 +476,12 @@ class Indentation
     {
         readonly IndentedTextWriter writer;
         readonly int indent;
-        string s;
+        string? s;
 
         internal Indentation(IndentedTextWriter writer, int indent)
```

Comment:
`IndentedTextWriter` is not used, we can remove it.

Reply:
?, I tried, ConsoleLog is still using it.